binarySearch/binarySearch.js

```diff
@@ -12,4 +12,21 @@
 
 const binarySearch = (nums, target) => {
 
+	let highIndex = nums.indexOf(nums[nums.length - 1]);
+	let lowIndex = 0;
+
+
```

Comment:
Probably don't need all this space between your variables and your while loop. I'd remove at least one space maybe both, because linters will catch this type of stuff..